R/McmcOptions-class.R

```diff
@@ -14,9 +14,20 @@ NULL
 #' @slot step (`count`)\cr only every `step`-th iteration is saved after
 #'   the `burnin`. In other words, a sample from iteration
 #'   `i = 1,...,iterations`, is saved if and only if
-#'   `(i - burnin) mod step = 0`. For example, for `iterations = 6`, 
-#'   `burnin = 0` and `step = 2`, only samples from iterations `2,4,6`
-#'   will be saved.
+#'   `(i - burnin) mod step = 0`.\cr
+#'   For example, for `iterations = 6`, `burnin = 0` and `step = 2`, only
+#'   samples from iterations `2,4,6` will be saved.
+#' @slot RNG (`list`)\cr a Random Number Generator (RNG) type and seed used
```

Comment:
agree with your idea to better make this two slots - easier for the user and we should further abstract here from what internally is needed and passed to JAGS.